internal/dataplane/sendconfig/dbmode.go

```diff
@@ -72,6 +72,7 @@ func (s UpdateStrategyDBMode) Update(ctx context.Context, targetContent ContentW
 		KongClient:      s.client,
 		SilenceWarnings: true,
 		IsKonnect:       s.isKonnect,
+		IncludeLicenses: true,
```

Comment:
Does `Syncer` omit `Licenses` when it's in Konnect mode?

Reply:
Yes. In GDR, the licenses are omitted when `IsKonnect` is `true`:
https://github.com/Kong/go-database-reconciler/blob/a20fb98749720aad87cbc17ab496e07be6593ca0/pkg/diff/diff.go#L201-L203